api/websocket/server/server.go

```diff
@@ -247,6 +248,16 @@ func (ws *WsServer) Stop() {
 
 //websocketHandler
 func (ws *WsServer) websocketHandler(w http.ResponseWriter, r *http.Request) {
+	host, _, err := net.SplitHostPort(r.RemoteAddr)
+	if err == nil {
```

Comment:
Do we need to check if `err != nil` ?

Reply:
In theory it shouldn't happen, and even it does, we can't do anything about it except for print it...